packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -2504,10 +2500,10 @@ class InteractiveGraph extends React.Component<Props, State> {
                 rubric.correct.type === "point" &&
                 userInput.coords != null
             ) {
-                let correct = InteractiveGraph.getPointCoords(
-                    rubric.correct,
-                    component,
-                );
+                let correct = rubric.correct.coords;
+                if (correct == null) {
+                    throw new Error("Point graph rubric has null coords");
+                }
```

Comment:
Maybe this is fine, but I want to ask @jeremywiebe. This validation logic is moving to the back-end and I'm not sure we want to be throwing or...[[insert idea here]]. Because if this throws on the BE, the client gets a 500 unless we handle it cleverly. I wonder if it would be better to return "invalid" (or report the error and return a correct answer since this would be KA's fault?)

Reply:
I see Jeremy's out today. I don't think this needs to block the PR, we can circle back to it.

Reply:
It's possible that the work I'm going to do pursuant to [this slack thread on WidgetOptions validation](https://khanacademy.slack.com/archives/C067UM1QAR4/p1727307152791499) will solve this. Our parser can default `coords` to `[]` if it's not present. It should also log an error if it has to set any defaults, so we can at least detect the problem.

Or maybe it would be better to default `correct` to a special value like `"ungraded"`, so the learner isn't penalized? That would require changing this logic.

Reply:
I guess my feeling is: do the bare minimum to remove `component` and if we think we need to circle back we can file a ticket.

Reply:
I think we'll need to handle invalid data coming from the frontend no regardless of this situation so I don't think this too bad throwing here. We'll also want to wrap whatever our final validation/scoring logic in error handling to avoid the server just crashing when an error is thrown. 

To the specific point, it looks like many of the other graph types access the `rubric.correct.coords` optimistically so this `point` type now does something extract. I'm not sure it's worth it unless we apply the same check/logic to all other types that access `rubric.correct.coords`.